qiskit/assembler/assemble_circuits.py

```diff
@@ -138,7 +203,19 @@ def assemble_circuits(circuits, run_config, qobj_id, qobj_header):
     qobj_config.memory_slots = max(memory_slot_sizes)
     qobj_config.n_qubits = max(qubit_sizes)
 
-    experiments = parallel_map(_assemble_circuit, circuits)
+    experiments_and_pulse_libs = parallel_map(_assemble_circuit, circuits, [run_config])
+    experiments = []
+    pulse_library = {}
+    for exp, lib in experiments_and_pulse_libs:
```

Comment:
Will this fail if `None` is returned from `_assemble_pulse_gates`?

Reply:
no, line 245 handles that now